main.go

```diff
@@ -20,17 +20,21 @@ import (
 )
 
 var opts struct {
-	Output       string
-	InputDir     string
-	TemplateFile string
-	Versions     []string
+	Output           string
+	InputDir         string
+	TemplateFile     string
+	IssueRegex       string
+	PullRequestRegex string
+	Versions         []string
 }
 
 func init() {
 	pflag.StringVarP(&opts.InputDir, "input", "i", "changelog", "read input files from `dir`")
 	pflag.StringVarP(&opts.Output, "output", "o", "", "write generated changelog to this `file` (default: print to stdout)")
 	pflag.StringVarP(&opts.TemplateFile, "template", "t", filepath.FromSlash("changelog/CHANGELOG.tmpl"), "read template from `file`")
 	pflag.StringSliceVar(&opts.Versions, "version", nil, "only print `version` (separate multiple versions with commas)")
+	pflag.StringVarP(&opts.IssueRegex, "regex-issue", "", `/.*/.*/issues/(\d+)`, "regex to parse issue urls from scm `expr`")
+	pflag.StringVarP(&opts.PullRequestRegex, "regex-pr", "", `/.*/.*/pull/(\d+)`, "regex to parse pull request urls scm `expr`")
```

Comment:
Should probably be URLs instead of urls? It's an acronym after all :)

Reply:
Also, I'd argue that it's more natural to name these options `issue-regex` and `pr-regex`, it's just more natural to what you'd call them in general. Also in the code you used IssueRegex instead of RegexIssue, I guess that validates it. The only reason to keep it as it is now would be to list them together in Cobra output I suppose.

Reply:
May I suggest the following descriptions?

~~~
`regex` for parsing SCM pull requests URLs
`regex` for parsing SCM issue URLs
~~~

Using `regex` instead of `expr` for the value is more to the point, an "expression" is less specific than a "regular expression".

Reply:
You're right, the only reason to name the flags `regex-*` was to keep them displayed together in the help output :) I've added your suggestions.